### PR TITLE
Elastic blast

### PR DESCRIPTION
**Pull request by r78v10a07**
This is a new recipe. For elastic-blast https://blast.ncbi.nlm.nih.gov/doc/elastic-blast/

### CODE REVIEW

File: recipes/elastic-blast/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - awscli
+    - awslimitchecker
+    - tenacity
+    - google-cloud-sdk
+    - importlib-metadata
+
+test:
+  commands:
+    - elastic-blast.py --version
+
```
Comment (christiam) [inline recipes/elastic-blast/meta.yaml:37]:
```suggestion
    - elastic-blast.py --help
    - elastic-blast.py submit --dry-run --loglevel DEBUG --logfile stderr --program blastn --db nt --aws-region us-east-1 --results s3://dummy-bucket --query /dev/null
```

File: recipes/elastic-blast/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - tenacity
+    - google-cloud-sdk
+    - kubernetes-client ==1.18.8
+    - importlib-metadata
+
+test:
+  commands:
+    - elastic-blast.py --version
+    - elastic-blast.py --help
+    - elastic-blast.py submit --dry-run --loglevel DEBUG --logfile stderr --program blastn --db nt --aws-region us-east-1 --results s3://dummy-bucket --query /dev/null
```
Comment (christiam) [inline recipes/elastic-blast/meta.yaml:39]:
```suggestion
    - elastic-blast.py submit --dry-run --results gs://dummy-bucket --gcp-project my-gcp-project --gcp-zone us-east4-b --gcp-region us-east4 --db pdbnt --program blastn --query /dev/null
```

File: recipes/elastic-blast/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - awslimitchecker
+    - tenacity
+    - google-cloud-sdk
+    - kubernetes-client ==1.18.8
+    - importlib-metadata
+
+test:
+  commands:
+    - elastic-blast.py --version
+    - elastic-blast.py --help
```
Comment (christiam) [inline recipes/elastic-blast/meta.yaml:38]:
```suggestion
    - elastic-blast.py --help
    - gcloud --version
    - gsutil --version
    - aws --version
    - kubectl version --client=true
```

**Review by christiam — CHANGES_REQUESTED** [review]

**Review by christiam — APPROVED** [review]

**Review by christiam — APPROVED** [review]

**Review by ypriverol — APPROVED** [review]

**Comment by r78v10a07** [comment]
@BiocondaBot please merge

**Pull request merged**